lnst/Recipes/ENRT/UseVfsMixin.py

```diff
@@ -12,19 +13,24 @@ class UseVfsMixin:
     with VF Device instances. This allows user to interact with the network
     interfaces without additional changes to the code of recipe.
 
+    Mixin provides two parameters:
+    * use_vfs - main boolean parameter to enable or disable (default) use of VFs
+    * vf_trust - (optional) set the trust parameter of the used VFs, 'on' or 'off'
+
     There are some limitations, for example pause frames cannot be configured
     since the VF do not support these.
     """
 
     use_vfs = BoolParam(default=False)
+    vf_trust = ChoiceParam(choices={'on', 'off'})
 
     def test_wide_configuration(self):
         config = super().test_wide_configuration()
 
         if not self.params.use_vfs:
             return config
 
-        config.vf_config = {}
+        self.vf_config = {}
```

Comment:
why the change from `config` to `self`? i would imagine that we may want to export this value into our measurement result metadata to discern between the two possibilities and in that case it should probably be in the shared config object

Reply:
I changed this to be able to access the vf_config in `test_wide_description()`. I can add this to both `config.vf_config()` and `self.vf_config()`

Reply:
Err, and when I checked the code now I see the config is passed as argument. I'll double check that, most likely you have a valid point.

Reply:
I changed this because I need this in https://github.com/LNST-project/lnst/blob/1d72d11f371312ac835c373b1d539e4757bf32eb/lnst/Recipes/ENRT/BondRecipe.py#L252.

To have the device list as property I need to store this in recipe object directly.

Reply:
Latest version does not require this anymore, so I removed this change.